ansible/roles/jenkins-worker/templates/freebsd.initd.j2

```diff
@@ -28,8 +28,8 @@ jenkins_env=" \
   NODE_TEST_DIR=/home/{{ server_user }}/tmp \
   PATH=/usr/local/libexec/ccache:/usr/local/bin:${PATH} \
   JOBS={{ server_jobs|default(ansible_processor_vcpus|default('2')) }} \
-  CC=cc \
-  CXX=c++"
+  CC={{ custom_cc_command|default('cc') }} \
```

Comment:
Yeah, I think a more complex change is necessary to do it "right" and my change wasn't quite there.  I've updated the PR to use your suggestion as it's the most straightforward approach.